libs/npm/core/src/components/grid-table/GridTable.tsx

```diff
@@ -158,8 +158,8 @@ export const GridTable = <T extends object>({
     useFilters,
     useGlobalFilter,
     useGroupBy,
-    useExpanded,
     useSortBy,
+    useExpanded,
```

Comment:
Fixing bug that broke all tables @jdtoombs

Reply:
Thanks :)